src/i18n/locales/en.json

```diff
@@ -0,0 +1,13 @@
+{
+  "signIn": "Sign In ",
+  "signOut": "Sign Out",
+  "Home": "Home",
+  "Claims": "Claims",
```

Comment:
Property names should be camel-cased. Also, keys need to be grouped. The keys mentioned here do not seem to be relevant to this ticket.